### PR TITLE
UI: Change Browser Panel to Person Panel

### PR DESCRIPTION
**Pull request by nicholaschuayunzhi**
This PR changes the `BrowserPanel` into a `PersonPanel`

### PersonPanel
- On `PersonPanelSelectionChangedEvent`, `PersonPanel` will display the tags, phone, address and email of the selected person.

### Feedback required
- Should we delete the `BrowserPanel`?
-  When do I update the user guide for the new look?
- I need help in making `PersonPanel` more attractive.
- On load, what should the `PersonPanel` show?

### Notable changes
- Changed `DarkTheme.css` to a white `main.css`.
- Removed phone, address and email from `PersonListCard`.

### TODO
- A main theme with jfx css variables
- Animations

### CODE REVIEW

File: src/main/java/seedu/address/ui/PersonCard.java [hunk 1]
```diff
@@ -54,9 +48,9 @@ public PersonCard(ReadOnlyPerson person, int displayedIndex) {
      */
     private void bindListeners(ReadOnlyPerson person) {
         name.textProperty().bind(Bindings.convert(person.nameProperty()));
-        phone.textProperty().bind(Bindings.convert(person.phoneProperty()));
-        address.textProperty().bind(Bindings.convert(person.addressProperty()));
-        email.textProperty().bind(Bindings.convert(person.emailProperty()));
+        //phone.textProperty().bind(Bindings.convert(person.phoneProperty()));
+        //address.textProperty().bind(Bindings.convert(person.addressProperty()));
+        //email.textProperty().bind(Bindings.convert(person.emailProperty()));
```
Comment (goweiwen) [inline src/main/java/seedu/address/ui/PersonCard.java]: Removing the lines is fine, no need to comment. We can refer to revision history if we need it back.

File: src/test/java/guitests/HelpWindowTest.java [hunk 2]
```diff
@@ -29,10 +29,10 @@ public void openHelpWindow() {
         getMainMenu().openHelpWindowUsingAccelerator();
         assertHelpWindowOpen();
 
-        getBrowserPanel().click();
+        getPersonPanel().click();
         getMainMenu().openHelpWindowUsingAccelerator();
-        assertHelpWindowNotOpen();
-
+        assertHelpWindowOpen();
+
```
Comment (goweiwen) [inline src/test/java/guitests/HelpWindowTest.java]: Remove trailing whitespace

File: src/test/java/guitests/guihandles/MainWindowHandle.java [hunk 3]
```diff
@@ -44,8 +44,9 @@ public StatusBarFooterHandle getStatusBarFooter() {
     public MainMenuHandle getMainMenu() {
         return mainMenu;
     }
-
-    public BrowserPanelHandle getBrowserPanel() {
-        return browserPanel;
+
```
Comment (goweiwen) [inline src/test/java/guitests/guihandles/MainWindowHandle.java]: Remove trailing whitespace

**Review by goweiwen — APPROVED** [review]
Code looks good. Any screenshots of the changed UI?

A few points:
- Remove trailing whitespaces
- Commented lines can be removed (we can refer to the commit if we need them back)

**Comment by nicholaschuayunzhi** [comment]
Thanks for the review, I've updated the code style accordingly. Please refer to `6562871` for UI.

**Review by hanselblack — APPROVED** [review]
Good changed to the CSS colour scheme.

**Pull request merged**